plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs

```diff
@@ -355,6 +354,7 @@ valueOf value cur tn =
     case Map.lookup tn tokens of
       Nothing -> 0
       Just v  -> v
+{-# INLINABLE valueOf #-}
 
 {-# INLINEABLE withCurrencySymbol #-}
```

Comment:
⚡ should be moved down.

Reply:
I think that script doesn't take into account alternative spelling INLINEABLE

Reply:
> I think that script doesn't take into account alternative spelling INLINEABLE

It does, but I think the problem was that in this file (and sometimes in other places) there was a blank line between the pragma and the thing it applied to, and I didn't consider that case.